core/adminserver/admin_server.go

```diff
@@ -82,17 +82,19 @@ var (
 )
 
 type server struct {
-	storage adminstorage.Storage
-	client  tpb.TrillianAdminClient
-	keygen  keys.ProtoGenerator
+	storage   adminstorage.Storage
+	logClient tpb.TrillianAdminClient
```

Comment:
yeah you've camelCased these, I think you should on the earlier ones too.

Reply:
Yep. done.